src/main/java/frc/team166/robot/Robot.java

```diff
@@ -54,6 +55,9 @@ public void robotInit() {
      */
     @Override
     public void disabledInit() {
+        drive.reset();
+        // lift.reset();
```

Comment:
Why are we not including the lift as part of this?

Reply:
We don't actually create a lift object in robot yet.